helpdesk_type/models/helpdesk_ticket.py

```diff
@@ -11,5 +11,6 @@ class HelpdeskTicket(models.Model):
 
     @api.onchange("type_id")
     def _onchange_type_id(self):
-        self.team_id = False
-        self.user_id = False
+        for record in self:
```

Comment:
Why are you changing this? `onchange` are per definition executed per record

Reply:
Done